packages/runtime/Runtime.ml

```diff
@@ -9,6 +9,23 @@ let fail_impossible_action_in_ssr fn =
 
 Here's the raw callstack:
 
+%s
+|}
+      fn raw_callstack
+  in
+  raise (Impossible_in_ssr (Printf.sprintf {|'%s' shouldn't run on the server|} fn))
+
+let fail_impossible_action_in_ssr_anonymous fn =
```

Comment:
How is this different than the other function? The only difference I can see is that this one prints the function with `Function:` while the other just shows it at the beginning of the msg (`'%s' sohld only run on the client...)